commands.json

```diff
@@ -2615,7 +2615,7 @@
   },
   "SREM": {
     "summary": "Remove one or more members from a set",
-    "complexity": "O(N) where N is the number of members to be removed.",
+    "complexity": "O(1) for each element removed, so O(N) to remove N elements when the command is called with multiple arguments.",
```

Comment:
Shouldn't this use "member" instead of "element"? Also needs to be standardized in `SADD`, `SDIFF` & `SDIFFSTORE`, `SSCAN` (please also remove the redundant period), `SUNION` & `SUNIONSTORE`.